README.md

```diff
@@ -1,6 +1,6 @@
 # Requirements
 
-* A ruby version manager such as [rvm](https://rvm.io/rvm/install) or [rbenv](https://formulae.brew.sh/formula/rbenv)
+* A ruby version manager such as [rvm](https://rvm.io/rvm/install), [rbenv](https://formulae.brew.sh/formula/rbenv) or [asdf](https://asdf-vm.com/guide/getting-started.html)
```

Comment:
Is it possible to keep PRs focused on one issue? It will make it easier to find relevant changes in the future.

Reply:
Thanks for the comment @AbuMareBear!  I agree with you -- one issue per PR.  
In this case since it was such a small README change I told DevelopMan in slack it was fine to put these together.  So that's on me.  

But I **really** appreciate you reminding folks about this, it's a lot to keep up with too.  It's helpful if we need to request a change on something we can merge everything else.